browser/brave_ads/ads_service_impl.cc

```diff
@@ -123,6 +123,8 @@ const unsigned int kRetriesCountOnNetworkChange = 1;
 
 constexpr char kAdNotificationUrlPrefix[] = "https://www.brave.com/ads/?";
 
+const base::Feature kAdServing{"AdServing", base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
this should go in a feature_list in the brave ads component

Reply:
@mkarolin @tmancey will handle this

Reply:
Leaving it to @tmancey for a follow-up